config/locales/en.yml

```diff
@@ -1,4 +1,12 @@
 en:
+  week_days:
+    sunday: Sunday
+    monday: Monday
+    tuesday: Tuesday
+    wednesday: Wednesday
+    thursday: Thursday
+    friday: Friday
+    saturday: Saturday
```

Comment:
Please remove these and use `I18n.t('date.day_names')`, because this will use the Rails supplied translations, meaning it is available in other languages as well.